packages/moonstone/FormCheckboxItem/FormCheckboxItem.module.less

```diff
@@ -24,6 +24,15 @@
 		overflow: hidden;
 	}
 
+	.itemIcon {
+		display: inline-block;
```

Comment:
Might want to use `display: contents` here. Noticing that the left margins collapse in 3.0.0 but not any longer with the current code and this fix.

nebula:

![image](https://user-images.githubusercontent.com/788456/64457468-88845300-d0a7-11e9-8a8a-de37bcece28f.png)

this:

![image](https://user-images.githubusercontent.com/788456/64457542-b8cbf180-d0a7-11e9-90e3-73a645e4542e.png)